app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,22 @@
+class SessionsController < ApplicationController
+  def new; end
```

Comment:
```suggestion
  // GET: /login
  def new; end
```
Thêm các comment router phía trên action cho tường minh code e nhé